test/table.html

```diff
@@ -0,0 +1,705 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>Table Sample</title>
+	<link href='https://fonts.googleapis.com/css?family=Lato:400,300' rel='stylesheet' type='text/css'>
```

Comment:
You can import the font file from the `test.scss` file so you just need to include one CSS file.